uavcan/node/435.ExecuteCommand.0.1.uavcan

```diff
@@ -0,0 +1,68 @@
+#
```

Comment:
what do you think about calling this SystemCall?

Reply:
It should be a verb, so `ExecuteSystemCall`? It is a higher-level action, not really a system call, isn't it?